localstripe/resources.py

```diff
@@ -2878,3 +2887,41 @@ def __init__(self, card=None, customer=None, **kwargs):
 
         self.type = 'card'
         self.card = card_obj
+
+
+class Balance(StripeObject):
+    object = 'balance'
+
+    def __init__(self, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        # All exceptions must be raised before this point.
+        super().__init__()
+
+        currencies = ['usd', 'eur']
```

Comment:
This list is arbitrary, and `usd` was never used in localstripe before. Do you need it, or can we keep `eur` for consistency?